src/BladeUIKit.php

```diff
@@ -57,6 +57,10 @@ public static function outputScripts(bool $force = false): string
             ->sort(fn (string $script) => str($script)->contains('alpine') ? 1 : 0)
             ->map(function (string $script) {
                 if (str($script)->contains('alpine')) {
+                    if (config('livewire.inject_assets') && !app()->runningUnitTests()) {
+                        return;
+                    }
```

Comment:
Heya, could you explain why you needed to add this?

Reply:
Because if we have Livewire installed it will inject Alpine JS automatically on every page, so there is not need to include Alpine script if `inject_assets` is true.

The Dusk tests use Livewire and won't work if Alpine JS in not loaded, so I think in testing environment it's ok to have the Alpine script.

Reply:
Perfect, thanks @alexmanase